bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/ComponentUtils.java

```diff
@@ -166,4 +177,103 @@ public static ContentPolicy getPolicy(String contentPath, ResourceResolver resou
         return policy;
     }
 
+    /**
+     * Takes in a single variable and returns an array of size 1 consisting of the same variable.
+     * Useful to directly iterate on, when something returns either a single value or an array of values of a particular type.
+     *
+     * @param param of any type T
+     * @return array of size 1 having the same variable
+     */
+    public static <T> T[] convertToArray(T param) {
+        if (param.getClass().isArray()) {
+            return (T[]) param;
+        } else {
+            T[] array = (T[]) Array.newInstance(param.getClass(), 1);
```

Comment:
Can we not use reflection package here, doesn't seem to be a usecase of reflection. Can't we simply create an array of size 1.